### PR TITLE
enable writing solution for multiple time steps

### PR DESCRIPTION
**Pull request by ashesh2512**

### CODE REVIEW

File: src/ExaTioga.cpp [hunk 1]
```diff
@@ -34,6 +34,8 @@ ExaTioga::ExaTioga(stk::ParallelMachine& comm)
     , m_stk(comm)
     , m_amr()
     , m_tioga(tioga_nalu::TiogaRef::self().get())
+    , m_doWrite(true)
+    , m_outputFreq(1)
```
Comment (sayerhs) [inline src/ExaTioga.cpp]:
```suggestion
```

File: src/ExaTioga.h [hunk 2]
```diff
@@ -32,14 +32,17 @@ class ExaTioga
...
 
     stk::ParallelMachine m_comm;
 
     tioga_nalu::StkIface m_stk;
     TiogaAMRIface m_amr;
 
     TIOGA::tioga& m_tioga;
+
+    bool m_doWrite;
+    int m_outputFreq;
```
Comment (sayerhs) [inline src/ExaTioga.h]:
```suggestion

    int m_output_freq{1};
    bool m_do_write{true};
```

Comment (ashesh2512) [reply src/ExaTioga.h]: @sayerhs This has be rectified in 
eac195f

File: src/ExaTioga.cpp (hunk 1 above)
Comment (ashesh2512) [reply src/ExaTioga.cpp]: @sayerhs This has be rectified in 
eac195f

**Pull request merged**